README.md

```diff
@@ -16,14 +16,27 @@ Pre-requisites: `oc`, `jq` and `git` should be pre-installed and you should be l
 
 ```bash
 # STEP 1: Install Dev Spaces next
-git submodule init && git submodule update && git -C devspaces checkout devspaces-3-rhel-8 &&
-cd devspaces/product && ./installDevSpacesFromLatestIIB.sh --next && \
-oc patch OperatorHub cluster --type json -p '[{"op": "add", "path": "/spec/disableAllDefaultSources", "value": false}]' # re-enable default catalog sources
+DSC_HOME=/home/user/.dsc
+TEMP_DIR="$(mktemp -d)"
+DSC_VERSION="3.4.0"
+DSC_ARCH="linux-x64"
+DSC_TGZ="devspaces-${DSC_VERSION}-dsc-${DSC_ARCH}.tar.gz"
```

Comment:
you could simplify this script by simply piping the curl to tar so you don't need a filename and don't need to delete the temporary tarball.